### DISCUSSION TITLE
How is duration computed for long main thread tasks?

### DISCUSSION THREAD

**Discussion by darahak-datadome** [GoogleChrome/lighthouse#13779] [Q&A] [answered]
Hi there,

I'm using Lighthouse in Chrome DevTools and I focus on the Performance report that signals long main thread tasks.
Inside the report page, I press the `View Original Trace` so I can identify what is causing those long tasks, but the numbers don't seem to match.

For example, Lighthouse will report a task that is running for more than 70ms.
When I look at the charts from the original trace, the longest task is only about 20ms and there is nothing in the `Long tasks` chart either.

Could someone clarify how the report should be interpreted and if there is an issue with the results?
Thank you.

**Comment by connorjclark** [comment, marked as answer]
see https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#devtools-lighthouse-panel-throttling

short answer: Lighthouse does a simulation by default, that trace is not expected to match (that's why it's called _Original_ Trace)
